the-collab-lab/tcl-64-smart-shopping-list
# Tailwind Setup

_For an example of how to fill this template out, [see this Pull Request](https://github.com/the-collab-lab/tcl-3-smart-shopping-list/pull/44)._

## Description



- Followed setup Install [Tailwind CSS with Vite](https://tailwindcss.com/docs/guides/vite)
- Added our project's main color to tailwind.config.js
- Added styling to h1 h2 as an example. Feel free to change this. 

## Acceptance Criteria
- Check if TailwindCss works on your end.


## Type of Changes



|     | Type                       |
| --- | -------------------------- |
|     | :bug: Bug fix              |
| ✓   | :sparkles: New feature     |
|     | :hammer: Refactoring       |
|     | :100: Add tests            |
|     | :link: Update dependencies |
|     | :scroll: Docs              |

## Updates
![Screenshot from 2023-09-29 10-28-51](https://github.com/the-collab-lab/tcl-64-smart-shopping-list/assets/73622805/547da13c-5051-41d7-956f-d1cb81a7eb85)






## Testing Steps / QA Criteria
- After `npm install` and `npm start`, you can check if the smart shopping List on the Homepage is green or not.
